src/wallet/transaction.h

```diff
@@ -129,44 +211,12 @@ class CWalletTx
         nTimeSmart = 0;
         fFromMe = false;
         fChangeCached = false;
-        fInMempool = false;
```

Comment:
There appear to be remaining `fInMempool` comments that may need updating.

```cpp
src/wallet/wallet.cpp:1733:    // We must set fInMempool here - while it will be re-set to true by the
src/wallet/wallet.cpp-1734-    // entered-mempool callback, if we did not there would be a race where a
src/wallet/wallet.cpp-1735-    // user could call sendmoney in a loop and hit spurious out of funds errors
src/wallet/wallet.cpp-1736-    // because we think that this newly generated transaction's change is
src/wallet/wallet.cpp-1737-    // unavailable as we're not yet aware that it is in the mempool.
src/wallet/wallet.cpp-1738-    //
src/wallet/wallet.cpp:1739:    // Irrespective of the failure reason, un-marking fInMempool
src/wallet/wallet.cpp-1740-    // out-of-order is incorrect - it should be unmarked when
src/wallet/wallet.cpp-1741-    // TransactionRemovedFromMempool fires.
--
src/wallet/wallet.cpp:1973    // Get the inserted-CWalletTx from mapWallet so that the
src/wallet/wallet.cpp:1974    // fInMempool flag is cached properly
```